solutions/Stepsproblem.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+using namespace std;
+bool Stair(int n)
+{
+	int rem5=n%5,rem7=n%7;
+	if(rem5%7==0||rem7%5==0)

```

Comment:
This Logic fails. Suppose n = 12 then first 7 steps and then 5 steps will take to 12.

Reply:
Change required